src/3d/qgsrubberband3d.h

```diff
@@ -88,10 +92,15 @@ class _3D_EXPORT QgsRubberBand3D
       /**
        * A circle is used to highlight points (○)
        */
-      Circle
+      Circle,
+
+      /**
+       * Points are not highlighted, neither the edges connecting them
```

Comment:
:thinking: why would marker type == None affect also edge highlighting? that's a bit unexpected